SUMMARY.md

```diff
@@ -0,0 +1,33 @@
+# Table of contents
+
+* [Welcome to BOLT, Friend!](README.md)
+
+## Read
+
+* [BOLT #0: Introduction](00-introduction.md)
+* [BOLT #1: Messaging](01-messaging.md)
+* [BOLT #2: Peer Protocol](02-peer-protocol.md)
+* [BOLT #3: Transactions](03-transactions.md)
+* [BOLT #4: Onion Routing](04-onion-routing.md)
+* [BOLT #5: Onchain](05-onchain.md)
+* [BOLT #6: DEPRECATED](https://github.com/lightningnetwork/lightning-rfc/issues/551)
+* [BOLT #7: Routing Gossip](07-routing-gossip.md)
+* [BOLT #8: Transport](08-transport.md)
+* [BOLT #9: Features](09-features.md)
+* [BOLT #10: DNS Bootstrap](10-dns-bootstrap.md)
+* [BOLT #11: Payment Encoding](11-payment-encoding.md)
+
+## Contribute
+
+* [Contribution Guide](CONTRIBUTING.md)
+
+## Proposals
+
+* [Proposal States](https://github.com/lightningnetwork/lightning-rfc/wiki/Lightning-Specification-1.1-Proposal-States)
```

Comment:
I'm not sure this is kept up-to-date. For the proposals maybe I'd list to the open PRs instead:
https://github.com/lightningnetwork/lightning-rfc/pulls

Reply:
Can do @cdecker @renepickhardt wdyt?

Reply:
I'd simply remove this since indeed this doesn't seem to be maintained very much (and clicking on "GitHub", then PRs should be a no-brainer).

Reply:
Removed.